tests/smoke/.makim-ssh.yaml

```diff
@@ -0,0 +1,43 @@
+hosts:
+  test_container:
+    host: localhost
+    port: 2222
+    user: testuser
+    password: testpassword
```

Comment:
@abhijeetSaroha , 

it would be nice if these values could be rendered from the jinja2 template, so it could accept env variables.

have you tested this use case?

Reply:
Not now, I will test and update you later.